centos-ci/run-rdgo

```diff
@@ -5,12 +5,13 @@ basedir=$(cd $(dirname $0) && pwd)
 . ${basedir}/libtask.sh
 
 cd rdgo
-ln -sf ${buildscriptsdir}/overlay.yml .
+ln -sfr ${buildscriptsdir}/overlay.yml .
```

Comment:
Won't this symlink be broken inside the container?
Sounds like this should be `/srv/src/overlay.yml`?

Reply:
But why does it work for FAHC?   Anyways, yeah might as well just do that.

Reply:
FAHC has

Reply:
Ugh, how did I miss that?  I must have been looking that the wrong copy of the file?  Anyways done :arrow_down: and it looks like it's working.